linux/devdoc/error_handling_linux_requirements.md

```diff
@@ -7,30 +7,29 @@ Linux implentaton of GetLastError and SetLastError
 ## Exposed API
 
 ```c
-MOCKABLE_FUNCTION(, void, error_handling_linux_set_last_error, volatile_atomic int64_t, err_code);
-MOCKABLE_FUNCTION(, uint64_t, error_handling_linux_get_last_error);
+MOCKABLE_FUNCTION(, void, error_handling_linux_set_last_error, uint32_t, err_code);
+MOCKABLE_FUNCTION(, uint32_t, error_handling_linux_get_last_error);
 ```
 
 ### set_last_error
 
 ```c
-MOCKABLE_FUNCTION(, void, error_handling_linux_set_last_error, volatile_atomic int64_t, err_code);
+MOCKABLE_FUNCTION(, void, error_handling_linux_set_last_error, volatile_atomic int32_t, err_code);
 ```
 `error_handling_linux_set_last_error` implements the PAL wrapper for the SetLastError function from Windows.
 
-`error_handling_linux_set_last_error` shall assign a non-`NULL` value to `last_error_code`.
+**SRS_ERROR_HANDLING_LINUX09_002: [** `error_handling_linux_set_last_error` shall assign a non-`NULL` value to `last_error_code`. **]**
```

Comment:
>a non-`NULL` [](http://example.com/codeflow?start=89&length=12)

These are not pointers, so just say this:

shall assign the value to `last_error_code`.